library/src/main/scala/scala/scalajs/runtime/RuntimeString.scala

```diff
@@ -46,7 +46,7 @@ private[runtime] object RuntimeString {
 
   def codePointAt(thiz: String, index: Int): Int = {
     val high = thiz.charAt(index)
-    if (index+1 < thiz.length) {
+    if (index + 1 < thiz.length) {
```

Comment:
No this is an undesirable (but tolerated) change. Arithmetic operations on the side of a comparison operator should not have spaces around it, to highlight the relative priority. This is filed as https://github.com/olafurpg/scalafmt/issues/511